src/test/java/org/broadinstitute/hellbender/tools/spark/validation/CompareBaseQualitiesIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.tools.spark.validation;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class CompareBaseQualitiesIntegrationTest extends CommandLineProgramTest {
+    @Test
+    public void test() throws Exception {
+        final String resourceDir = getTestDataDir() + "/spark/validation/";
+
+        final File firstBam = new File(resourceDir, "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
+        final File secondBam = new File(resourceDir, "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam");
```

Comment:
Do you intend for `firstBam` and `secondBam` to be the same for this test? Shouldn't one be a GATK3 bam, and the other a GATK4 bam created by running BQSR within this test? Is this test class just a temporary stub?